<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="1" failed="0" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-05-13T06:41:03 PDT" name="Default suite" finished-at="2022-05-13T06:41:05 PDT" duration-ms="1451">
    <groups>
    </groups>
    <test started-at="2022-05-13T06:41:03 PDT" name="Default test" finished-at="2022-05-13T06:41:05 PDT" duration-ms="1451">
      <class name="com.qa.api.testcases.POST_User_Scenario13_InvalidPathInUrl">
        <test-method signature="postWithValidData()[pri:0, instance:com.qa.api.testcases.POST_User_Scenario13_InvalidPathInUrl@ef9296d]" started-at="2022-05-13T06:41:03 PDT" name="postWithValidData" finished-at="2022-05-13T06:41:05 PDT" duration-ms="1408" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- postWithValidData -->
      </class> <!-- com.qa.api.testcases.POST_User_Scenario13_InvalidPathInUrl -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
